components/Seo.tsx

```diff
@@ -17,6 +17,12 @@ const sites = {
     description:
       'Drei is a growing collection of useful helpers and abstractions for react-three-fiber.',
```

Comment:
We may need to look over Drei stuff to fix outdated references to R3F or update to V6. Might create an issue if this is too extensive to do now.

Reply:
In storybook or in the docs?

Reply:
Both maybe, but that's a separate issue. I'll note it in Discord and take a look.